parser/parser_test.go

```diff
@@ -335,19 +335,21 @@ func (s *testParserSuite) TestParser0(c *C) {
 
 		// For select with where clause
 		{"SELECT * FROM t WHERE 1 = 1", true},
+
+		// For comment in query
+		{"/*comment*/ /*comment*/ select c /* this is a comment */ from t;", true},
```

Comment:
It is better to add different types of comments.
like 
`/* xxx */ select * from t`
`select * /* xxx */ from t;`
`select * from t /* xxx */`